api/test.go

```diff
@@ -0,0 +1,42 @@
+package api
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/vocdoni/arbo"
+	"github.com/vocdoni/arbo/memdb"
+	"github.com/vocdoni/vocdoni-z-sandbox/state"
+)
+
+var tree *arbo.Tree
+var testAvailable bool
+
+func init() {
+	if os.Getenv("TEST_API") == "true" || os.Getenv("TEST_API") == "1" {
+		var err error
+		tree, err = arbo.NewTree(arbo.Config{
+			Database:     memdb.New(),
+			MaxLevels:    state.MaxLevels,
+			HashFunction: state.HashFunc,
+		})
+		if err != nil {
+			panic(err)
+		}
+		testAvailable = true
+	}
+}
+
+func (a *API) setProcessInfoForTest(w http.ResponseWriter, r *http.Request) {
+	if !testAvailable {
```

Comment:
Instead of doing this check I'd just not register the endpoint if the tests are not enabled.